src/test/java/core/basesyntax/service/ParserServiceTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserServiceTest {
+    private ParserService parserServiceTest;
+    private List<String> testListLines;
+    private List<Transaction> expected;
+    private List<Transaction> actual;
+
+    @Before
+    public void setUp() {
+        actual = new ArrayList<>();
+        expected = new ArrayList<>();
+        testListLines = new ArrayList<>();
+        parserServiceTest = new ParserServiceImpl();
+    }
+
+    @Test
+    public void parseEmptyLine_notOk() {
```

Comment:
```suggestion
    public void parseEmptyLine_ok() {
```
this is correct behaviour, right?